src/Input.test.jsx

```diff
@@ -0,0 +1,38 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import Input from './Input';
+
+import { TASK_TITLE } from './fixture';
+
+describe('Input', () => {
+  const handleChangeTitle = jest.fn();
+  const handleClickAddTask = jest.fn();
+
+  const renderInputComponent = () => render(
```

Comment:
```suggestion
  const renderInput = () => render(
```

`Component`라는 말은 없어도  될 것 같아요

Reply:
굳이 없어도 충분히 이해할 수 있다고 보시기 때문이겠지요?? 수정했습니다 :)